tests/unit/test_core/test_switch.py

```diff
@@ -68,13 +68,13 @@ def tearDown(self):
 
     def test_switch_vlan_pool_default(self):
         """Test default vlan_pool value."""
-        self.assertEqual(self.options.vlan_pool, '{}')
+        self.assertEqual(self.options.vlan_pool, {})
 
     def test_switch_vlan_pool_options(self):
         """Test switch with the example from kytos.conf."""
         dpid = "00:00:00:00:00:00:00:01"
-        vlan_pool_json = '{"00:00:00:00:00:00:00:01": ' \
-                         + '{"1": [[1, 2], [5, 10]], "4": [[3, 4]]}}'
+        vlan_pool_json = {"00:00:00:00:00:00:00:01":
+                          {"1": [[1, 2], [5, 10]], "4": [[3, 4]]}}
         self.controller.switches[dpid] = self.switch
         self.options.vlan_pool = vlan_pool_json
```

Comment:
As you changed the format, the variable name got outdated.

```suggestion
        vlan_pool = {"00:00:00:00:00:00:00:01":
                     {"1": [[1, 2], [5, 10]], "4": [[3, 4]]}}
        self.controller.switches[dpid] = self.switch
        self.options.vlan_pool = vlan_pool
```